spec/features/expense_category_show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe "Visit the show page of a user's expense category", type: :feature do
+  # Logged in before running the tests
+  before do
+    user = FactoryBot.create(:user)
+    login_as(user)
+    @expense_category = FactoryBot.create(:expense_category, user:)
+    @expense_category.icon.attach(io: File.open('spec/fixtures/files/avatar.jpg'), filename: 'avatar.jpg')
+    @expenses = FactoryBot.create_list(:expense, 3, author: user, expense_category: @expense_category)
```

Comment:
- [ ] Can you work on the testing again because the test failed to pass?
- 
![Capture](https://github.com/Grandi0z/cents_sense/assets/63972670/4915f15b-6d79-498f-ac86-75486fb5d8ce)